recipe/install_nvcc.sh

```diff
@@ -122,8 +122,14 @@ EOF
 
 # Create `nvcc` script in `bin` so it can be easily run.
 mkdir -p "${PREFIX}/bin"
-cat > "${PREFIX}/bin/nvcc" <<EOF
+cat > "${PREFIX}/bin/nvcc" <<'EOF'
 #!/bin/bash
-"\${CUDA_HOME}/bin/nvcc" -ccbin "\${CXX}" \$@
+for arg in "${@}" ; do
+  case ${arg} in -ccbin)
+    # If -ccbin argument is already provided, don't add an additional one.
+    exec "${CUDA_HOME}/bin/nvcc" "${@}"
+  esac
+done
+exec "${CUDA_HOME}/bin/nvcc" -ccbin "${CXX}" "${@}"
```

Comment:
If we find `-ccbin` in the arguments, will this execute `nvcc` twice? The first time within thise loop, and then again after the loop?

Reply:
`exec` replaces the current process (i.e., the wrapper script) with the actual `nvcc`.

Reply:
Yeah it shouldn't. Though agree it might be clearer if we had a simple `if...else` here.

Reply:
To me all of this looks very clear -- but well, I've just messed about with POSIX-y shell code a lot the last few years with all these packaging stuff...
If there are code comments or changes you'd like to have which would help comprehension for the maintainers, i.e., you, please add
````
```suggestion
```
````
blocks I can apply.
(If you're unsure about shell syntax, then I can also just translate pseudo code on the fly if you give me some.)

Reply:
TIL about `exec`. This is fine @mbargull, resolving this.